src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,15 @@
 package calculator;
 
+import calculator.controller.CalculatorController;
+import calculator.service.CalculatorService;
+import calculator.viiew.CalculatorView;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        CalculatorService calculatorService = new CalculatorService();
+        CalculatorView calculatorView = new CalculatorView();
+        CalculatorController calculatorController = new CalculatorController(calculatorService, calculatorView);
```

Comment:
```suggestion
CalculatorController calculatorController = new CalculatorController(new CalculatorService() , new CalculatorView());
```

[제안] 개인적으로 main에서 두 객체를 주입만 하기 때문에 이처럼 인라인으로 하는 것이 어떨까 싶어요!

Reply:
아! 그런 방법이 있었네요!! 다음번에는 한번 제안해주신것처럼 해봐야겠네요!